mrc/localization/position/probability.py

```diff
@@ -0,0 +1,28 @@
+import math
+
+from mrc.utils.maths import gauss_distribution
+
+
+class AngularProbability:
+    def __init__(self, stdev, mean, coef):
+        self.mean = mean
+        self.stdev = stdev
+        self.coef = coef
+
+    def __call__(self, rotation):
+        d = math.pi - self.mean
+        p1 = gauss_distribution(rotation, self.mean, self.stdev)
+        p2 = gauss_distribution(rotation, -math.pi - d, self.stdev)
```

Comment:
Are `p1 `and `p2 `supposed to be the same?

Reply:
`-math.pi - d` = `-math.pi - (math.pi - self.mean)` =` -math.pi - math.pi + self.mean` = `self.mean`

Reply:
My bad, it's not the same